### PR TITLE
update index.md

### PR DESCRIPTION
**Pull request by matthewjosef**
wording and markdown fixes.

### CODE REVIEW

File: src/pages/get-building/index.md [hunk 1]
```diff
@@ -4,9 +4,8 @@ title: Get Building
 
 In Quick Start you installed Holochain and built a templated forum application. Here you will use the Scaffold tool to build your own application and learn the basic commands for using the scaffold.
 
-!!! note Pre-requisite
-Holochain installed as per [Quick Start](../quick-start/index)
-!!! 
+| :exclamation:  note Pre-requisite: Holochain installed as per [Quick Start](../quick-start/index) |
```
Comment (marycamacho) [inline src/pages/get-building/index.md:7]: This change should be be reverted as the other code is using current markdown per the new spec.

File: src/pages/get-building/index.md [hunk 2]
```diff
@@ -99,23 +101,31 @@ You should then see:
...
 ```
 
-Press `y` again.
+Again, press `y`
 
-You will then see `Coordinator zome "todos" scaffolded!` along with output from the intial downloading and setting up of the Holochain rust hdk.  Followed by instructions for adding your first entry type.
+You will then see 
+```text
+Coordinator zome "todos" scaffolded! 
+```
```
Comment (marycamacho) [inline src/pages/get-building/index.md:127]: are you using the ``` full line of code snippet  instead of the `code words` inline form here on purpose?

**Review by marycamacho — CHANGES_REQUESTED** [review]
The first change should be reverted but I think the others will work fine. I did not do a cross browser test on this but will do so before we merge. Otherwise it's just my one other question about your intention on code snippets.

File: src/pages/get-building/index.md [hunk 3]
```diff
@@ -71,13 +69,17 @@ You should then see :
 ? DNA name (snake_case): 
 ```
 
-Many hApps have just one DNA, so in this case you can just type: `todos`
+Many hApps have just one DNA, so in this case you can just type: 
+```bash
```
Comment (pdaoust) [inline src/pages/get-building/index.md:73]:
```suggestion

```bash
```

File: src/pages/get-building/index.md [hunk 4]
```diff
@@ -99,23 +101,31 @@ You should then see:
  (The integrity zome will automatically be named '{name of coordinator zome}_integrity')
 ```
 
-Type in a name for the zome.  In this case we can just use the same name as the DNA `todos`
-You should then see: 
+Type in a name for the zome.  In this case we can just use the same name as the DNA
```
Comment (pdaoust) [inline src/pages/get-building/index.md:104]:
```suggestion
Type in a name for the zome.  In this case we can just use the same name as the DNA:

```

File: src/pages/get-building/index.md [hunk 5]
```diff
@@ -99,23 +101,31 @@ You should then see:
...
 ```
 
-Type in a name for the zome.  In this case we can just use the same name as the DNA `todos`
-You should then see: 
+Type in a name for the zome.  In this case we can just use the same name as the DNA 
 ```bash
+todos
+```
+
+You should then see:
```
Comment (pdaoust) [inline src/pages/get-building/index.md:109]:
```suggestion
You should then see:

```

File: src/pages/get-building/index.md [hunk 6]
```diff
@@ -99,23 +101,31 @@ You should then see:
...
 ```text
 Integrity zome "todo_integrity" scaffolded!
 ? Scaffold coordinator zome in "dnas/todos/zomes/coordinator/"? (y/n) ›
 ```
 
-Press `y` again.
+Again, press `y`
 
-You will then see `Coordinator zome "todos" scaffolded!` along with output from the intial downloading and setting up of the Holochain rust hdk.  Followed by instructions for adding your first entry type.
+You will then see
```
Comment (pdaoust) [inline src/pages/get-building/index.md:124]:
```suggestion
You will then see:

```

File: src/pages/get-building/index.md (hunk 2 above)
Comment (pdaoust) [inline src/pages/get-building/index.md:127]:
```suggestion
```

```

Comment (pdaoust) [reply src/pages/get-building/index.md:127]: ack, GitHub is getting confused. What I'm trying to do is add a newline after the `\`\`\``, not delete it entirely.

File: src/pages/get-building/index.md [hunk 7]
```diff
@@ -125,11 +135,14 @@ In our To-do happ every to-do item is stored as an entry so let's add new entry
 ```
 
 You should see: 
-```bash
+```text
```
Comment (pdaoust) [inline src/pages/get-building/index.md:138]:
```suggestion

```text
```

File: src/pages/get-building/index.md [hunk 8]
```diff
@@ -125,11 +135,14 @@ In our To-do happ every to-do item is stored as an entry so let's add new entry
...
 ✔ Entry type name (snake_case): ·
 ```
 
-type: `todo_item`
+type:
```
Comment (pdaoust) [inline src/pages/get-building/index.md:142]:
```suggestion
Type:

```

File: src/pages/get-building/index.md [hunk 9]
```diff
@@ -149,14 +162,19 @@ Which fields should the entry contain?
...
+
+So, use the up and down arrows to select `String` and press `<enter>`
 
 You should see: 
 ```text
 ? Field name: › 
 ```
 
-type `description`
+type
```
Comment (pdaoust) [inline src/pages/get-building/index.md:174]:
```suggestion
Type:

```

File: src/pages/get-building/index.md [hunk 10]
```diff
@@ -186,7 +204,7 @@ You should then see:
 ✔ Delete
 ```
 
-The scaffolding tool can add zome and UI functions for updating and deleting entries.  In the case of our todo app we want to be able to do both, which is the default, so just press  `<enter>`
+The scaffolding tool can add zome and UI functions for updating and deleting entries.  In the case of our todo app we want to be able to do both, which is the default, so just press `<enter>`
```
Comment (pdaoust) [inline src/pages/get-building/index.md:207]:
```suggestion
The scaffolding tool can add zome and UI functions for updating and deleting entries.  In the case of our todo app we want to be able to do both, which is the default, so just press `<enter>`.
```

File: src/pages/get-building/index.md [hunk 11]
```diff
@@ -213,7 +231,11 @@ You should then see:
 Collection name (snake_case, eg. "all_posts"): › 
 ```
 
-Type in: `my_todos` and press `<enter>`
+Type:
```
Comment (pdaoust) [inline src/pages/get-building/index.md:234]:
```suggestion
Type:

```

File: src/pages/get-building/index.md [hunk 12]
```diff
@@ -213,7 +231,11 @@ You should then see:
 Collection name (snake_case, eg. "all_posts"): › 
 ```
 
-Type in: `my_todos` and press `<enter>`
+Type: 
+```bash
+my_todos
+```
+and press `<enter>`
```
Comment (pdaoust) [inline src/pages/get-building/index.md:238]:
```suggestion

and press `<enter>`.
```

File: src/pages/get-building/index.md [hunk 13]
```diff
@@ -253,7 +275,7 @@ So lets follow those hints:
 import MyTodos from './todos/todos/MyTodos.svelte';
 import CreateTodoItem from './todos/todos/CreateTodoItem.svelte';
 ```
-2. Replace the "EDIT ME" text there with:
+2. Replace the "EDIT ME! Add the components of your app here." text (about halfway down the page of the App.svelte file) with:
```
Comment (pdaoust) [inline src/pages/get-building/index.md:278]:
```suggestion
2. Replace the "EDIT ME! Add the components of your app here." text (about halfway down the page of the App.svelte file) with:

```

File: src/pages/get-building/index.md [hunk 14]
```diff
@@ -268,8 +290,8 @@ You now have a fully functional Holochain app up and running!
 Next steps might include creating a collection of all ToDo items, not just the ones you created, and adding UI to see them.  Enjoy and get building!
 
 !!! learn Learn More ——>
-
 - Find all the links to Rust docs, HDKs and APIs in [References](../references/).
-- Seeing strange language in these steps look up the words in the [Glossary](../references/glossary.md).
-- Go deep to understand how Holochain works in the [Core Concepts](../concepts/).
-!!!
\ No newline at end of file
+- Seeing strange language in these steps look up the words in the [Glossary](../references/glossary.md). |
```
Comment (pdaoust) [inline src/pages/get-building/index.md:294]:
```suggestion
- Seeing strange language in these steps? Look up the words in the [Glossary](../references/glossary.md). |
```

**Comment by pdaoust** [comment]
This one is obsolete; there's a new guide.

**Pull request closed without merging**